pkg/util/globalerror/grpc.go

```diff
@@ -159,3 +142,9 @@ func (e ErrorWithStatus) Equals(err error) bool {
 	}
 	return false
 }
+
+func createCauses(errs ...error) []error {
+	causes := make([]error, 0, 2)
+	causes = append(causes, errs...)
+	return causes
```

Comment:
i don't understand this. Why don't we just return `errs`?

Reply:
I removed this code